src/main/java/in/projecteka/consentmanager/consent/ConsentManager.java

```diff
@@ -34,6 +40,10 @@
     private final ConsentRequestRepository consentRequestRepository;
     private final ConsentArtefactRepository consentArtefactRepository;
     private KeyPair keyPair;
+    @Autowired
```

Comment:
Could you please remove @Autowired annotation, remove constructor and add @AllArgsConstructor?